apps/examples/testcase/le_tc/network/tc_net_pkt.c

```diff
@@ -0,0 +1,146 @@
+/*  @file tc_net_socket.c
+ * @brief Test Case Example for pbuf() API
+ */
+
+#include <tinyara/config.h>
```

Comment:
This header is unnecessary. Let's remove.

Reply:
done